ChildProject/annotations.py

```diff
@@ -77,6 +78,13 @@ class AnnotationManager:
         'UC6': 'OCH'
     }
 
+    VTC_SPEAKER_TYPE_TRANSLATION = defaultdict(lambda: 'NA', {
```

Comment:
does this mapping seem alright to you ? @alecristia

Reply:
yes, other than those 4 changes (ELE+OCHI)